pallets/collective/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "pallet-collective"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
```

Comment:
Should be opentensor? Or at least adding opentensor

Reply:
Probably should add it, it’s partially modified